### ISSUE TITLE
Problem with 'use-subscription'

### ISSUE THREAD

**Issue by soundsnick** [artalar/reatom#407]
Had an issue while an attempt to run reatom v2.
Is there any solution for this?

My project specifications:
- React 17.0.2
- webpack 5.4.0
- No Typescript 

Error description:
```
ERROR in ../../../node_modules/@reatom/react/build/index.es.js 1:101-152
Module not found: Error: Can't resolve 'use-subscription' in '\www\project\node_modules\@reatom\react\build'
resolve 'use-subscription' in '\www\project\node_modules\@reatom\react\build'
  Parsed request is a module
  using description file: \www\project\node_modules\@reatom\react\package.json (relative path: ./build)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      \www\project\node_modules\@reatom\react\build\node_modules doesn't exist or is not a directory
      \www\project\node_modules\@reatom\react\node_modules doesn't exist or is not a directory
      \www\project\node_modules\@reatom\node_modules doesn't exist or is not a directory
      \www\project\node_modules\node_modules doesn't exist or is not a directory
      looking for modules in \www\project\node_modules
        single file module
          using description file: \www\project\package.json (relative path: ./node_modules/use-subscription)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              \www\project\node_modules\use-subscription doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              \www\project\node_modules\use-subscription.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              \www\project\node_modules\use-subscription.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              \www\project\node_modules\use-subscription.wasm doesn't exist
        \www\project\node_modules\use-subscription doesn't exist
 @ consume shared module (default) @reatom/react@^2.0.15 (strict) (fallback: ../../../node_modules/@reatom/react/build/index.es.js)
 @ ./src/components/features/demo-state/component.jsx 2:0-40 6:17-24
 @ ./src/routes.jsx 3:0-71 45:46-55
 @ ./src/App.jsx 5:0-34 10:192-198
 @ container entry ./EpkApp[0]

webpack 5.70.0 compiled with 1 error in 3597 ms
i ｢wdm｣: Failed to compile.

```

**Comment by kompolom** [comment]
You should install use-subscription manually

```
npm i --save use-subscription
```

**Comment by artalar** [comment]
Hi, I am working on removing `use-subscription` as a dependency. Found a few bugs, make a refactoring. I will publish an update in a couple of days.

**Comment by soundsnick** [comment]
@kompolom thanks, it helped

**Comment by soundsnick** [comment]
@artalar any updates on this?

**Comment by artalar** [comment]
@soundsnick will publish new version of react package tomorrow.

**Comment by artalar** [comment]
Fixed in `2.0.18`
